Common/Orders/Order.cs

```diff
@@ -285,5 +285,9 @@ public static Order CreateOrder(SubmitOrderRequest request)
             }
             return order;
         }
+        /// <summary>
+        /// Order Expiry on a specific UTC time.
+        /// </summary>
+        public DateTime DurationValue;
```

Comment:
yes! #awesome! It should default to DateTime.MaxValue and we can scan it. Later we can actually implement the GTC / Day options! 

Potentially we should also include the Enum value here since if sending in *real* orders to brokerages you'd want to use their GTC / Day settings and not fake it with LEAN. 

To avoid scope creep I vote leaving it as "Expiry" (no enum) & addressing timed expiration scanning / sending-live-orders properly in a new issue.